README.md

```diff
@@ -14,7 +14,7 @@ Please ask questions on [Stackoverflow](http://stackoverflow.com/questions/tagge
 How do I, as a developer, start working on the project?
 
 1. cd final  (or phase1)
-1. `mvn clean appengine:devserver`
+1. `mvn clean appengine:run` OR `./gradlew appengineRun`
```

Comment:
I prefer `gradle appengineRun` and a mention of ./gradlew.sh for folks who don't have Gradle installed.  But I'm Ok w/ this if you insist.